packages/expo-polyfills/package.json

```diff
@@ -42,6 +43,7 @@
     "esbuild": "0.19.5",
     "esbuild-node-externals": "1.9.0",
     "jest": "29.7.0",
+    "jest-expo": "^49.0.0",
```

Comment:
Why `jest-expo`?  I don't see any calls to it.  And it's big :(

Reply:
It's the test environment for Expo, it actually fixes a lot of problems and better represents the various environments for `Android`, `iOS`, and `Web` as separate environments. See the `jest.config.ts`: 
https://github.com/medplum/medplum/blob/2c3fe80d47b96bf23806bcaa236c3d6f9d7d4d28/packages/expo-polyfills/jest.config.json#L2